.github/workflows/pypi-package.yml

```diff
@@ -0,0 +1,22 @@
+name: Building & Publishing
+
+on:
+  push:
+    tags: ["v*"]
```

Comment:
I specified that this workflow should only trigger after tags starting with `v` are pushed (i.e. `v0.10.0`)